src/wp-includes/blocks.php

```diff
@@ -497,10 +497,21 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 		$metadata
 	);
 
-	return WP_Block_Type_Registry::get_instance()->register(
+	$block = WP_Block_Type_Registry::get_instance()->register(
```

Comment:
Is this the only path for block registration? I see that there's a `register_block_type` that does not always call `register_block_type_from_metadata`.

Reply:
I've moved the action to the class, so it looks like all paths for block registration should be covered now.